index.php

```diff
@@ -4,63 +4,107 @@
 
 function prepDirectoryPath($file) {
 
-    $path = explode('/', $file);
+    //ensure that the file is not empty
+    if($file!=='' &&!empty($file) && isset($empty))
```

Comment:
Should be good on the `isset` since, unlike JS, PHP will throw an warning if that parameter isn't passed. Used this as a test to verify that `empty` would still work as expected:

```php
<?php

function thing($var) {
  var_dump(empty($var));
}

thing();
```

Output:

```
PHP Warning:  Missing argument 1 for thing(), called in /mnt/sdb1/www/brakkit/reflect.php on line 7 and defined in /mnt/sdb1/www/brakkit/reflect.php on line 3
PHP Stack trace:
PHP   1. {main}() /mnt/sdb1/www/brakkit/reflect.php:0
PHP   2. thing() /mnt/sdb1/www/brakkit/reflect.php:7
bool(true)
```